packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -210,14 +210,41 @@ async function getNpmTarballUrl(packageName: string, version: string = 'latest')
   return stdout.trim();
 }
 
+/**
+ * Gets expo SDK version major from the local package.json.
+ */
+async function getLocalSdkMajorVersion(): Promise<string | null> {
+  const path = require.resolve('expo/package.json', { paths: [process.cwd()] });
+  if (!path) return null;
+  const { version } = require(path) ?? {};
+  return version?.split('.')[0] ?? null;
+}
+
+/**
+ * Selects correct version of the template based on the SDK version for local modules and EXPO_BETA flag.
+ */
+async function getTemplateVersion(targetDir: string, isLocal) {
```

Comment:
Interesting that linter doesn't catch this 🤔 
```suggestion
async function getTemplateVersion(isLocal: boolean) {
```

Reply:
Apparently it's a TS message, but not a build error, but the linter doesn't pick it up
```
'targetDir' is declared but its value is never read.ts(6133)
```